src/xdocs/checks/header/regexpheader.xml

```diff
@@ -100,13 +100,8 @@ line 6: ^\W*$
               <td>charset</td>
               <td>Specify the character encoding to use when reading the headerFile.</td>
               <td><a href="../../property_types.html#String">String</a></td>
-              <td>
-                <code>
-                  the charset property of the parent<a href="../../config.html#Checker">
-                  Checker</a> module
-                </code>
-              </td>
-              <td>5.0</td>
+              <td><code>the charset property of the parent &lt;a href=&quot;https://checkstyle.org/config.html#Checker&quot;&gt;Checker&lt;/a&gt; module</code></td>
+              <td>4.4</td>
```

Comment:
Same.

Reply:
all since are reverted. map is updated with values from xdoc